lib/core/rancher/step.sh

```diff
@@ -32,7 +32,22 @@ function ih::setup::core.rancher::install() {
 
     cp "${THIS_DIR}/io.rancherdesktop.profile.defaults.plist" "$HOME/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
 
-    if command -v rdctl; then
+    # If rancher is already installed  reset to factory
+    if [ -d "/opt/homebrew/Caskroom/rancher" ] || [ -d "/opt/homebrew/Caskroom/ih-rancher" ] 
```

Comment:
This path is different on M1 and Intel. Better to just use `brew list ih-rancher`. It will have a non-zero exit code if there is no such formula/cask installed.